packages/@aws-cdk/aws-lambda-nodejs/lib/bundling.ts

```diff
@@ -208,7 +244,11 @@ export class Bundling implements cdk.BundlingOptions {
           throw new Error(`Expected esbuild version ${ESBUILD_MAJOR_VERSION}.x but got ${Bundling.esbuildInstallation.version}`);
         }
 
-        const localCommand = createLocalCommand(outputDir, Bundling.esbuildInstallation);
+        if (!Bundling.tscInstallation) {
+          throw new Error('Unable to find typescript locally or globally make sure install typescript in you dependencies');
```

Comment:
We cannot throw here if typescript is not available but precompliation was not requested.